### ISSUE TITLE
Application specific addition (openHAB)

### ISSUE THREAD

**Issue by ThomDietrich** [willemdh/FireMotD#46]
@willemdh just wanted to ask before I invest any effort. Would you consider merging openHAB specific statistics? (version, opstate, overall devices and rules defined... could be a oneliner)
Obviously that's pretty specific for one application and might exceed the scope and maintainability of the one FireMotD script file...

**Comment by willemdh** [comment]
@ThomDietrich I absolutely have no problem with that. As long as you provide the necessasry if variable exists => show variable code
And also when I ever have the time and energy to convert FireMotD so i saves all the information in a .json file, you help me port this.

Fyi, I'm heavily invested in Nagios and Elasticsearch and it's on my to do to add the Nagios Core, Elasticsearch, Logstash and Kibana versions too one day.

**Comment by ThomDietrich** [comment]
Hey,
sure thing! I totally get your point. With this confirmation I'll look into this hopefully soon.

Nagios is also part of my setup (although not on the systems I'm using FireMotD on) and I've worked with the ELK stack for quite some time (not anymore currently). Both would be great additions.

Regarding the json: Could you elaborate? You want to generate all information upfront (e.g. via cron) and only show it upon login?

**Comment by willemdh** [comment]
> Regarding the json: Could you elaborate? You want to generate all information upfront (e.g. via cron) and only show it upon login?

Correct, this has been on my to do for quite some time. Instead of using /var/tmp/updatecount.txt i would like to write to a .json file which contains all the info. On logon, only uptime, cpu usage, cpu load, memory, swap and disk usage, sessions and processes would have to be generated.

The big question is if and when I'll have time for this of course... 

About the git flow options, I'll think about it and experiment with the pr's a bit. Again when I find some spare time, not making any promises.. :)

**Comment by willemdh** [comment]
@ThomDietrich The new JSON configuration file should allow you to easily add custom checks which could even require sudo permissions.

**Comment by ThomDietrich** [comment]
Amazing!
I'm still willing to do these additions and with the new config this might become easier 👍 
This will take me a few more weeks to look into...

**Comment by willemdh** [comment]
@ThomDietrich I made an example with Elasticsearch version.
1. Add your custom addtion to FireMotD.json in FireMotD
2. Create Explore function
3. Regenerate FireMotD.json
4. Add conditionals to theme functions

Feel free to comment if you have any ideas making it easier
